### PR TITLE
intermediate_source /per_sample_grads.py 번역 완료

### PR DESCRIPTION
**Pull request by NamsanMan**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #1115 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

intermediate_source /per_sample_grads.py 의 내용을 번역 완료하였습니다.

### CODE REVIEW

**Comment by rlarbdus128-sys** [comment]
우선 전반적으로 이해하기 쉽게 정말 잘 번역하신 것 같습니다! 수고하셨습니다.

아래에, 제가 읽으며 수정을 제안드리고 싶은 부분을 간단히 적어보았으니 오탈자 PR 생성에 참고하셔도 좋을 것 같습니다.

line 6: 의미 전달을 위해 원문대로 의문문 문말에 '?'을 붙여도 좋을 것 같습니다.

line 149, 181: 표준어 규정에 따라 소괄호와 단어를 띄어쓰는 것보다 문맥에 맞게 붙여쓰는 것이 적합할 것으로 생각됩니다. 다른 line에서는 잘 지켜진 사항이기에 의도하신 부분인지 확인 후 수정해주시길 바랍니다.

line 99, 100, 116, 131, 143, 144, 149, 150, 151, 153, 163, 170, 173, 179, 212, 215, 217: 백틱 외부 띄어쓰기는 표준어 규정에 따르므로, 백틱과 조사 사이 띄어쓰기를 삭제하는 것이 적합할 것으로 보입니다. 다만 다른 한글화 문서를 보았을 때 가시성을 위해 백틱과 뒤따르는 단어를 띄어쓰는 것이 널리 허용되는 면이 있고 'PyTorch Korea 한국어 튜토리얼 번역 가이드라인' 등을 조사해보니 마땅한 규정이 없는 것으로 보입니다. 온전히 오탈자로 보기 어려우나 여지가 있으므로 멘토님께 여쭤보고 추가 수정을 할지 결정해보면 좋을 것 같습니다.

다시 한 번 정말 수고 많으셨습니다. 감사합니다.

**Comment by NamsanMan** [comment]
꼼꼼한 리뷰 감사합니다! 해당 내용 점검 후 다른 리뷰가 오면 적극 반영해서 같이 수정해 보도록 하겠습니다!

**Comment by Junhyun17** [comment]
번역하느라 고생 많으셨습니다.
꼼꼼하게 잘 번역하셔서 수정하실 부분도 거의 없으신것 같습니다.

위의 리뷰와 겹치지 않게 수정사항을 제안 드리자면,

line 4, 7에서 '===', '---' 부분이 원문과 다르게 바뀌어서  수정하시면 좋을 거 같습니다.

line 150에서 '계산하게 할 수 있습니다.' 부분을 '계산할 수 있습니다.'로 고치시면 조금 더 문장이 자연스러울거 같습니다.

번역하느라 수고 많으셨습니다!!

**Comment by NamsanMan** [comment]
리뷰 남겨 주셔서 감사합니다! 해당 부분 반영해서 수정 진행하도록 하겠습니다!

File: intermediate_source/per_sample_grads.py [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # -*- coding: utf-8 -*-
 """
-Per-sample-gradients
+표본별 변화도
 ====================
```
Comment (hyoyoung) [inline intermediate_source/per_sample_grads.py:5]: 저자 표현이 없더라도, 역자 표현 추가를 권장드립니다.
다른 문서보시고 역자 표기를 넣어주시기 바랍니다

File: intermediate_source/per_sample_grads.py [hunk 2]
```diff
@@ -98,41 +96,40 @@ def compute_sample_grads(data, targets):
...
-# We'll use the ``torch.func.functional_call`` function to treat an ``nn.Module``
-# like a function.
+# ``torch.func.functional_call`` 함수를 사용하여 ``nn.Module`` 을 함수처럼 다룹니다.
 #
-# First, let’s extract the state from ``model`` into two dictionaries,
-# parameters and buffers. We'll be detaching them because we won't use
-# regular PyTorch autograd (e.g. Tensor.backward(), torch.autograd.grad).
+# 먼저 ``model``의 상태를 parameters와 buffers라는 두 딕셔너리로 추출합니다.
+# 일반적인 PyTorch autograd(예: Tensor.backward(), torch.autograd.grad)는 사용하지 않으므로
+# 이 값을 detach합니다.
```
Comment (hyoyoung) [inline intermediate_source/per_sample_grads.py]: 이 값을 분리(detach)합니다
라고 의역해도 좋을거 같습니다

File: intermediate_source/per_sample_grads.py [hunk 3]
```diff
@@ -98,41 +96,40 @@ def compute_sample_grads(data, targets):
...
 
 ######################################################################
-# Next, let's define a function to compute the loss of the model given a
-# single input rather than a batch of inputs. It is important that this
-# function accepts the parameters, the input, and the target, because we will
-# be transforming over them.
+# 다음으로 입력 배치가 아니라 단일 입력이 주어졌을 때
+# 모델의 손실을 계산하는 함수를 정의하겠습니다.
+# 이 함수는 매개변수, 입력, target을 인자로 받아야 합니다.
+# 변환을 이 인자들에 대해 적용할 예정이기 때문입니다.
```
Comment (hyoyoung) [inline intermediate_source/per_sample_grads.py]: 이 인자들을 기준으로 변환을 적용할 예정
이라고 변경해도 좋을거 같습니다

**Review by hyoyoung — CHANGES_REQUESTED** [review]
전반적으로 매끄러운 번역하시느라 수고하셨습니다.
몇가지 수정사항 요청드리니 확인부탁드립니다

File: intermediate_source/per_sample_grads.py (hunk 1 above)
Comment (NamsanMan) [reply intermediate_source/per_sample_grads.py:5]: "표본별 변화도(Per-sample gradients)"로 수정했습니다!

File: intermediate_source/per_sample_grads.py (hunk 2 above)
Comment (NamsanMan) [reply intermediate_source/per_sample_grads.py]: 수정 사항 반영했습니다. 실수로 conversation을 resolve 처리했는데, 해당 코멘트 반영 완료 상태입니다. 확인 부탁드립니다.

File: intermediate_source/per_sample_grads.py (hunk 3 above)
Comment (NamsanMan) [reply intermediate_source/per_sample_grads.py]: 수정완료하였습니다!

**Review by hyoyoung — CHANGES_REQUESTED** [review]
다른 문서 보시고 역자 혹은 번역 항목을 추가해서
본인 이름과 깃헙 주소를 넣어주시길 부탁드립니다.
문서 관리를 위해 저자 항목이 없더라도 번역자 항목은 필수로 넣어두고 있습니다

**Comment by NamsanMan** [comment]
comment 감사합니다! 말씀해주신 대로 추가하였습니다. 감사합니다.

**Review by hyoyoung — APPROVED** [review]
LGTM

**Review by jih0-kim — APPROVED** [review]
LGTM
